manager/manager.go

```diff
@@ -366,6 +433,11 @@ func (m *Manager) InitWithOptions(elf io.ReaderAt, options Options) error {
 		return err
 	}
 
+	// Remove excluded sections
+	for _, excludedSection := range m.options.ExcludedSections {
+		delete(m.collectionSpec.Programs, excludedSection)
```

Comment:
I believe this will cause matchSpecs to fail because the defined Probes will not have a matching program in the spec.

Reply:
Nice catch, I added a test in `matchSpecs` to handle excluded sections.